stable/papertrail/templates/daemonset-fluentd.yaml

```diff
@@ -0,0 +1,76 @@
+{{- if eq .Values.daemonset.type "fluentd" }}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name:  {{ template "papertrail.fullname" . }}
+  labels:
+    app: {{ template "papertrail.name" . }}
+    chart: {{ template "papertrail.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "papertrail.name" . }}
+        release: {{ .Release.Name }}
+      {{- if .Values.daemonset.podAnnotations }}
+      annotations:
+{{ toYaml .Values.daemonset.podAnnotations | indent 8 }}
+      {{- end }}
+    spec:
+      {{- if .Values.daemonset.tolerations }}
+      tolerations:
+{{ toYaml .Values.daemonset.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.daemonset.affinity }}
+      affinity:
+{{ toYaml .Values.daemonset.affinity | indent 8 }}
+      {{- end }}
+      {{- if .Values.daemonset.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.daemonset.nodeSelector | indent 8 }}
+      {{- end }}
+      volumes:
+        - name: varlog
+          hostPath:
+            path: {{ .Values.papertrail.fluentd.varLogDir }}
+        - name: varlibdockercontainers
+          hostPath:
+            path: {{ .Values.papertrail.fluentd.varLibDockerContainersDir }}
+        - name: config
+          configMap:
+            name: fluentd-papertrail-config
```

Comment:
The `ConfigMap` has the name of `name: {{ template "papertrail.fullname" . }}-config` so this will fail.

Also, even if the right `ConfigMap` name is configured, the chat will fail because fluentd needs a `Role` a `ServiceAccount` and a `RoleBinding`:
```
2019-03-19 09:39:25 +0000 [info]: parsing config file is succeeded path="/fluentd/etc/fluent.conf"
2019-03-19 09:39:27 +0000 [error]: config error file="/fluentd/etc/fluent.conf" error_class=Fluent::ConfigError error="Exception encountered fetching metadata from Kubernetes API endpoint: pods is forbidden: User \"system:serviceaccount:default:default\" cannot list pods at the cluster scope ({\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\"pods is forbidden: User \\\"system:serviceaccount:default:default\\\" cannot list pods at the cluster scope\",\"reason\":\"Forbidden\",\"details\":{\"kind\":\"pods\"},\"code\":403}\n)"
```

Related issue https://github.com/fluent/fluentd-kubernetes-daemonset/issues/87